backend/shared/createTransactionViewModel.js

```diff
@@ -1,14 +1,18 @@
 const { formatCurrency, formatTimestampToDate } = require('../utils')
 
-module.exports = (transactions, prisons, showBalance = true) =>
+module.exports = (transactions, prisons, showBalance = true, mergeAmounts = false) =>
   transactions.map(transaction => {
     const { description: prisonName } = prisons.find(agency => transaction.agencyId === agency.agencyId)
     const { penceAmount, currentBalance } = transaction
 
     return [
       { text: transaction.entryDate && formatTimestampToDate(transaction.entryDate) },
-      { text: transaction.postingType === 'CR' ? formatCurrency(penceAmount / 100) : '' },
-      { text: transaction.postingType === 'DR' ? formatCurrency(penceAmount / 100) : '' },
+      ...(mergeAmounts
+        ? [{ text: (penceAmount && formatCurrency(penceAmount / 100)) || '' }]
```

Comment:
I would have thought that `formatCurrency` should be able to cope with `penceAmount` being `0` or `false`?  Is it that if the value is `0` you don't want to display anything

Reply:
Agreed. Will improve it in the next PR.